### PR TITLE
TE-322 Form responsive

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-322)

### What **one** thing does this PR do?
Adds responsive styles for Form collection.

### Any other notes
Tweaks examples which consume `Form`

screen shot 2018-04-23 at 13 33 10

### CODE REVIEW

File: src/semantic/src/themes/livingstone/collections/form.overrides [hunk 1]
```diff
@@ -78,6 +82,13 @@
 --------------------*/
 
   /* Grouped Fields */
+  &[class*="equal width"] .fields > .field,
+  [class*="equal width"].fields > .field {
+    // This prevents any field from extending beyond the right edge of its
+    // containing field group. Flex sizing of each field is delegated to
+    // `semantic/src/definitions/collections/form.less` line 950
+    width: 0%;
```
Comment (davidpelayo) [inline src/semantic/src/themes/livingstone/collections/form.overrides]: ? I don't get it. Why `width: 0%`?

File: src/semantic/src/themes/livingstone/globals/site.variables [hunk 2]
```diff
@@ -133,6 +133,14 @@
      Breakpoints
 --------------------*/
 
+@mobileBreakpoint: 1px;
+@tabletBreakpoint: 600px;
+@computerBreakpoint: 1000px;
+@largeMonitorBreakpoint: 1920px;
+@widescreenMonitorBreakpoint: 1920px;
+
+@mobileScreen: ~"only screen and (min-width: @{mobileBreakpoint}) and (max-width: @{largestMobileScreen})";
```
Comment (davidpelayo) [inline src/semantic/src/themes/livingstone/globals/site.variables]: I believe this is not right?
Mobile breakpoint usually stands for the breakpoint in which lower values defines the mobile layout.
Why then the need of declaring a min-width of 1px?

Comment (Jonnygwi) [reply src/semantic/src/themes/livingstone/globals/site.variables]: +1

File: src/semantic/src/themes/livingstone/collections/form.overrides (hunk 1 above)
Comment (marcdavi-es) [reply src/semantic/src/themes/livingstone/collections/form.overrides]: It allows the children of an `equal width` field group to play nicely with `flex`. They grow to fill the available space.

Comment (marcdavi-es) [reply src/semantic/src/themes/livingstone/collections/form.overrides]: Sadly no. I know this seems hacky but it's the most reliable thing we can do.

**Pull request merged**